src/test/unit/routes/callback.test.ts

```diff
@@ -0,0 +1,120 @@
+import axios, { AxiosResponse } from 'axios';
```

Comment:
Could you double check all routes tests are working by running **yarn test:routes** I'm seeing an error on type of claim which you may have fixed

Reply:
As discussed, I have added the redirect tests to type of claim controller test.